blog/2017-01-07-monthly-release-cadence.md

```diff
@@ -0,0 +1,25 @@
+---
+title: A new Release Cadence: Releasing December and January RC
+author: Eric Vicenti
+authorTitle: Developer Advocate at Facebook
+authorURL: https://twitter.com/EricVicenti
+authorImage: https://secure.gravatar.com/avatar/077ad5372b65567fe952a99f3b627048?s=128
+authorTwitter: EricVicenti
+category: announcements
+---
```

Comment:
The frontmatter here all looks good, but to be sure, did you check that the blog post is properly rendered in the website? Is your author byline showing up properly?